contents/uix/design-systems.md

```diff
@@ -0,0 +1,108 @@
+3. `Not suitable for apps that emphasize on performance`: Front-end frameworks come with elements that the team may not need. The unused code negatively affects the overall performance.
+
+<box type="info">
+Note that the style guide mentioned here is a term used in the past when Design Systems have not been explored yet. The Style Guide of a Design System is more focused and emphasises on graphic styles only.
+</box>
+
+Style guides present components that can be used quickly in mockups. This is an example of Tor's [style guide](https://styleguide.torproject.org/components/).  However, there are a few disadvantages:
+1. `Not easily traceable`: It is hard to trace where the components came from as different types of components may be built over time by different teams.
+2. `Not easily documented`: Most of the time, style guides lack documentation on what each component does. Also, too much documentation may decrease reusability of components because it is infeasible for designers/developers to go through alot of pages to find a component they wish to use.
+
+## Benefits of a Design System
+Design Systems combine the good parts of front-end frameworks and style guides.
+
+**Benefit 1: Shared Common language**
+
+Design Systems act as a [Single Source of Truth](https://en.wikipedia.org/wiki/Single_source_of_truth). This allows the whole organization to access changes and updates. Any member of a team would have complete documentation of design mock-ups at their fingertips. This results in less misunderstandings and much better implementation of designs.
+
+Design Systems also [elimate knowledge silos](https://moduscreate.com/blog/design-systems-and-how-your-company-benefits-from-them/). There is lesser risk of context gaps being formed because information is shared across teams.
+
+**Benefit 2: Reusablility**
+
+Design Systems provide a shared library of reusable components. When every component in a design system is reusable, it substantially decreases the amount of re-work.
+
+Teams can use these reusable components to quickly assemble and implement new pages, which means extra resources for focusing on more valuable things such as customer experience.
+
+**Benefit 3: Faster Development**
+
+When products are integrated with a Design System, updates would only be required to carried out in one place. This [blog post](https://moduscreate.com/blog/design-systems-and-how-your-company-benefits-from-them/) from Modus Create explains how Design Systems helped them to achieve faster development speed.
+
+In a [blog post](https://airbnb.design/building-a-visual-language/) from Airbnb, the team explained how they were able to create nearly 50 screens within just a few hours by using their Design System. It was also mentioned by the team that they build and release features on all native platforms at roughly the same time now. Development is faster for them now since product engineers can focus more on writing the feature logic rather than the view code.
```

Comment:
> rather than the view code.

did you mean viewing the code?

Reply:
It's quoted from their article. They meant the code that is responsible for the view layer (think MVC). 

But yes I agree it should be rephrased to something simpler or at least it should explain what view code means.